CIP-0043/README.md

```diff
@@ -0,0 +1,153 @@
+CIP: ?0043 
```

Comment:
I think you are missing the `---` before the first line for this to parse correctly.

Reply:
Indeed I am! This is because of a dirty merge.

Reply:
Please, rename the supporting folder and change the CIP number to `49` (without leading-zeros, otherwise Github markdown renderer has the brilliant idea to interpret those numbers in octal base :grimacing: ).